builtin/providers/aws/resource_aws_sns_topic_subscription.go

```diff
@@ -44,6 +45,24 @@ func resourceAwsSnsTopicSubscription() *schema.Resource {
 				Required: true,
 				ForceNew: false,
 			},
+			"endpoint_auto_confirms": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: false,
```

Comment:
Not everything you see elsewhere in the codebase is worth copying, yet many things do! :smiley: